src/grandorgue/GOrgueSettings.cpp

```diff
@@ -117,10 +117,12 @@ GOrgueSettings::GOrgueSettings(wxString instance) :
 	ReverbFile(this, wxT("Reverb"), wxT("ReverbFile"), wxEmptyString),
 	MemoryLimit(this, wxT("General"), wxT("MemoryLimit"), 0, 1024 * 1024, GOrgueMemoryPool::GetSystemMemoryLimit()),
 	SamplesPerBuffer(this, wxT("General"), wxT("SamplesPerBuffer"), 1, MAX_FRAME_SIZE, 1024),
-	SampleRate(this, wxT("General"), wxT("SampleRate"), 1000, 100000, 44100),
-	Volume(this, wxT("General"), wxT("Volume"), -120, 20, -15),
+    /*  SAMPLE RATE — Enabled 192000 by changing 100000 to 200000.
```

Comment:
This change seems doesn't belong to the Release Length / Truncation. Please remove it from here and make a separate PR for it.

Reply:
> This change seems doesn't belong to the Release Length / Truncation. Please remove it from here and make a separate PR for it.

Yes, that was an accidental push on my part, from my private build copy, sorry! It does work though and is worth implementing. If need be, I have 192 kHz samples you can test with.